platform/android/java/lib/patches/com.google.android.vending.expansion.downloader.patch

```diff
@@ -202,9 +202,9 @@ index e4b1b0f1c..36cd6aacf 100644
  
      public static String getSpeedString(float bytesPerMillisecond) {
 -        return String.format("%.2f", bytesPerMillisecond * 1000 / 1024);
-+        // -- GODOT start --
++        // -- BLAZIUM start --
 +        return String.format(Locale.ENGLISH, "%.2f", bytesPerMillisecond * 1000 / 1024);
```

Comment:
The line 205 maybe from a different branch, please validate this is required for rebranding